src/main/java/com/robosh/ejournal/entity/Student.java

```diff
@@ -18,17 +18,17 @@
 @EqualsAndHashCode(callSuper = true)
 @NoArgsConstructor
 @AllArgsConstructor
-@Entity
+@Entity(name = "student")
 public class Student extends User {
 
     @NotNull
     @JsonFormat(pattern="dd-MM-yyyy")
+    @Column(name = "date_of_birth")
     private LocalDate dateOfBirth;
 
     @OneToOne
     private Group group;
 
-    @ManyToMany
     private List<Parent> parents;
```

Comment:
Why did you remove this?

Reply:
because of double join table but it did not help anyway